testing/sharedtesthelpers/helpers.go

```diff
@@ -0,0 +1,236 @@
+package sharedtesthelpers
```

Comment:
Avoid using generic util libs - go/util-harmful

Can we move these into separate libs under /testing? e.g. one for error handling, one for package helpers, one for table testing.